### PR TITLE
Add method focus as a workaround for scroll issues

### PR DESCRIPTION
**Pull request by PhyNerd**

### CODE REVIEW

**Comment by Varun-Hegde** [comment]
https://user-images.githubusercontent.com/57041400/234375178-841ee63a-45a9-4f74-8270-452921fdb159.mp4

Will add tests in some time

File: test/reuse/ui5/control/focus.spec.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+"use strict";
+const { handleCookiesConsent } = require("../../../helper/utils");
+
+describe("focus - for a input field", function () {
```
Comment (DIFSRIP) [inline test/reuse/ui5/control/focus.spec.js:4]:
```suggestion
describe("control - focus - for a input field", function () {
```

File: test/reuse/ui5/control/focus.spec.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        "metadata": "sap.m.StandardListItem",
+        "bindingContextPath": "/entities/199"
+      }
+    };
+    await ui5.control.focus(selector);  //can see the scroll take place during this step
+    await ui5.assertion.expectToBeVisible(selector);
+  });
+});
+
+describe("focus - outside of viewpoint", function () {
```
Comment (DIFSRIP) [inline test/reuse/ui5/control/focus.spec.js:42]:
```suggestion
describe("control - focus - outside of viewpoint", function () {
```

File: test/reuse/ui5/control/focus.spec.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      "elementProperties": {
+        "viewName": "sap.m.sample.MultiComboBox.view.MultiComboBox",
+        "metadata": "sap.m.Token"
+      }
+    };
+    await ui5.assertion.expectAttributeToBe(selector, "text", "Smart Games");
+  });
+});
+
+describe("focus - no selector found and catch error", function () {
```
Comment (DIFSRIP) [inline test/reuse/ui5/control/focus.spec.js:79]:
```suggestion
describe("control - focus - no selector found and catch error", function () {
```

**Review by DIFSRIP — COMMENTED** [review]
Looks good so far. We can adjust the test titles for more transparency in the execution logs.  **(You can directly click the "commit suggestion" button. No need to change it manually)**
Also we could add some better error handling in the future.

**Review by DIFSRIP — APPROVED** [review]

**Pull request merged**